WinUIGallery/ControlPages/DesignGuidance/AccessibilityKeyboardPage.xaml

```diff
@@ -1,387 +1,673 @@
+    &lt;ListViewItem&gt;Green&lt;/ListViewItem&gt;
+    &lt;ListViewItem&gt;Yellow&lt;/ListViewItem&gt;
+&lt;/ListView&gt;
+                </x:String>
+            </core:ControlExample.Xaml>
+        </core:ControlExample>
+        
+        <TextBlock
+            Margin="0,20,0,0"
+            Style="{ThemeResource BodyStrongTextBlockStyle}"
+            AutomationProperties.HeadingLevel="Level3"
+            Text="Manually supporting arrow keys with XYFocusKeyboardNavigation" />
+
+        <RichTextBlock>
+            <Paragraph>
+                You can enable arrow key navigation between items manually, too.
+                <LineBreak />
+            </Paragraph>
+            <Paragraph>
+                Note that for true lists of items, users may expect additional affordances like
+                support for Home/End, PgUp/PgDn, and additional accessibility properties like
+                PositionInSet and SizeOfSet. Keyboard navigation can get complicated, but getting
+                it right can make your app a lot easier to use—for everyone.
+                <LineBreak />
+            </Paragraph>
+            <Paragraph>
+                See
+                <Hyperlink
+                    NavigateUri="https://learn.microsoft.com/en-us/windows/apps/design/input/focus-navigation">
+                    Focus navigation for keyboard, gamepad, remote control, and accessibility tools
+                </Hyperlink> and
+                <Hyperlink
+                    NavigateUri="https://learn.microsoft.com/en-us/windows/apps/design/accessibility/keyboard-accessibility">
+                    Keyboard accessibility
+                </Hyperlink>.
+            </Paragraph>
+        </RichTextBlock>
+
+        <core:ControlExample>
+            <core:ControlExample.Example>
+                <StackPanel Spacing="4">
+                    <StackPanel
+                        Background="{ThemeResource CardBackgroundFillColorSecondaryBrush}"
+                        BorderBrush="{ThemeResource SurfaceStrokeColorDefaultBrush}"
+                        AutomationProperties.Name="Potatoes?"
+                        BorderThickness="1"
+                        Spacing="8"
+                        Padding="8"
+                        CornerRadius="4">
+                        <TextBlock
+                            Style="{ThemeResource BodyStrongTextBlockStyle}"
+                            Text="Potatoes?" />
+                        <!--
+                            XYFocusKeyboardNavigation enables arrow keys between the buttons.
+
+                            Note that:
+
+                                - The buttons are still tabbable by default
+                                - Home/End and PgUp/PgDn do not work
+                                - Screen readers do not read PositionInSet/SizeOfSet ("1 of 3")
+
+                            All of that would require custom work.
+                        -->

```

Comment:
Do we need this comment in this code?

Reply:
I was thinking having it be present in the XAML sample code would be fine but I guess having it twice doesn't hurt and is also closer to reality when reading the sample code in app.